meshtastic/interfaces/ble/interface.py

```
@@ -605,64 +615,62 @@ def _on_ble_disconnect(self, client: BleakRootClient) -> None:
+        coordinator = self.thread_coordinator
+        dispatch_event = self._resolve_thread_event_dispatcher(coordinator)
+        if dispatch_event is not None:
+            dispatch_event(event_name)
             return
-        logger.debug(error_msg, exc_info=True)
+        logger.debug(
+            "No callable thread event dispatcher available for event %s",
+            event_name,
+        )
+
+    def _handle_malformed_fromnum(self, reason: str, exc_info: bool = False) -> None:
```

Ruff (0.15.6)

[warning] Boolean-typed positional argument in function definition

(FBT001)